controllers/gce/controller/utils.go

```diff
@@ -640,3 +673,34 @@ func (o PodsByCreationTimestamp) Less(i, j int) bool {
 	}
 	return o[i].CreationTimestamp.Before(o[j].CreationTimestamp)
 }
+
+// setInstanceGroupsAnnotation sets the instance-groups annotation with names of the given instance groups.
+func setInstanceGroupsAnnotation(existing map[string]string, igs []*compute.InstanceGroup) error {
```

Comment:
Recommend naming something multi-cluster specific

Reply:
The method does not have any multi cluster specific logic. Its a generic method that sets the instance groups annotation. Hence I am inclined to keep the name generic.

Reply:
Okay